staging/src/k8s.io/cloud-provider/controllers/route/route_controller.go

```diff
@@ -74,18 +74,13 @@ func New(routes cloudprovider.Routes, kubeClient clientset.Interface, nodeInform
 		klog.Fatal("RouteController: Must specify clusterCIDR.")
 	}
 
-	eventBroadcaster := record.NewBroadcaster()
```

Comment:
Did the panic risk get fixed for this as well?

Reply:
The k8s.io/cloud-provider/controllers/route controller is okay because it doesn't register informer callbacks and thus doesn't need the recorder unless `Run` is called.